### ISSUE TITLE
Smaller Tweak for 'Open How' dialogue

### ISSUE THREAD

**Issue by m1karii** [TeamNewPipe/NewPipe#8161]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

in the 'Open With' dialogue I'd like to change the 'only once' and 'always' buttons for 'always' checkbox + 'confirm' button. as I've multiple times accidentally missclicked the 'always' button and then had to search through the settings to revert it. this change will basically make the 'always' feature viable same as before but it will require 2 clicks instead of one preventing accidental triggering from happening

**Comment by TiA4f8R** [comment]
That's something which is not in our control I'm afraid. This dialog is shown by your Android system, which we can't customize.

The only option to do your behavior is to add a custom system app chooser, which is not something we plan to do at all, because of the niche improvement it will add (which is also not standard).

**Comment by SameenAhnaf** [comment]
Then, this is a duplicate of https://github.com/TeamNewPipe/NewPipe/issues/4102

**Comment by m1karii** [comment]
@SameenAhnaf @TiA4f8R 
> Then, this is a duplicate of #4102

no, it isn't. they sound similar but are actually very different. this is actually a duplicate of #1170 (which is mentioned in #4102 and has been closed aswell)

what person in #4102 is trying to achieve is: avoid 2 clicks in the menu. it either takes double-click on method of your choosing (e.g. play in the background, download,...) or one click to select the method + second click to choose 'just once'. this person is trying to avoid that by introducing 'just once' checkbox which would remember whether it has been checked from previous instance of the dialogue and you would only need to click once on the method of your choosing to execute it. (they also changed 'always' button to checkbox but it serves the same purpose as before)
I personally think that this is a really unintuitive and dumb idea (as it would introduce another problems). I also think the person did a really bad job of communicating it.

what me and person who opened #1170 are trying to achieve is remove annoying miss-clicks on 'always' button which then require us to search and change it in settings to 'always ask'. we try to achieve this by changing 'always' button to a checkbox and 'just once' button to a 'confirm' button as already explained above.

**Comment by SameenAhnaf** [comment]
#4102 still skips the hassle of double clicks. `Remember my checkbox` will always remain unchecked by default. 
You'll only click on it if you require to change the settings.

How often did you touch `Always` button accidentally?

**Comment by m1karii** [comment]
> How often did you touch `Always` button accidentally?

around once every month or two ... it is not every day, but still really annoying

**Comment by SameenAhnaf** [comment]
Ok, I suggested https://github.com/TeamNewPipe/NewPipe/issues/5940#issuecomment-818051629 which was initially intended to access video info and all buttons within the same menu. It's not expected to have any checkbox or something.

Maybe, this menu could be accessible for `Always ask` functionality instead?
